py4cast/datasets/base.py

```diff
@@ -629,14 +629,15 @@ def summary(self):
         print(f"Step_duration {self.step_duration}")
         print(f"Static fields {self.statics.grid_static_features.feature_names}]")
         print(self.statics.grid_static_features)
-        for p in ["forcing", "input_output", "diagnostic"]:
+        print(self.shortnames)
+        for p in ["input", "input_output", "output"]:
             names = self.shortnames[p]
             mean = self.stats.to_list("mean", names)
             std = self.stats.to_list("std", names)
             mini = self.stats.to_list("min", names)
             maxi = self.stats.to_list("max", names)
             units = [self.units[name] for name in names]
-            if p != "forcing":
+            if p != "input":  # Forcing variables
```

Comment:
Shouldn't it be **"inputs", "outputs", "forcing"** instead of ["input", "input_output", "output"]